src/js/utils/dom.js

```diff
@@ -21,7 +21,7 @@ import computedStyle from './computed-style';
  *
  */
 function isNonBlankString(str) {
-  return typeof str === 'string' && (/\S/).test(str);
+  return typeof str === 'string' && str.trim();
```

Comment:
not using regex is faster.

Reply:
Can a comment be added here explaining usage of trim()?

Reply:
Also, should we wrap `str.trim()` in a call to `Boolean`?

Reply:
yes